deployment/monocular/templates/api-deployment.yaml

```diff
@@ -23,13 +23,13 @@ spec:
           httpGet:
             path: /healthz
             port: {{ .Values.api.service.internalPort }}
-          initialDelaySeconds: 60
+          initialDelaySeconds: 180
           timeoutSeconds: 10
         readinessProbe:
           httpGet:
             path: /healthz
             port: {{ .Values.api.service.internalPort }}
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
```

Comment:
New PR here https://github.com/nomisbeme/monocular/pull/1
Or if you want this another way, let me know.